_sass/blocks/explore/_filters.scss

```diff
@@ -0,0 +1,47 @@
+
+.filters-wrapper {
+	padding: $base-padding;
+	margin-bottom: $base-padding;
+
+	@include respond-to(medium-up) {
+		border-radius: $base-border-radius;
+		margin-right: $base-padding;
+	}
+}
+
+.filters-heading {
+	margin-bottom: $base-padding;
```

Comment:
Line should be indented with spaces, not tabs<br>Properties should be ordered display, margin-bottom